internal/initwd/from_module_test.go

```diff
@@ -19,7 +19,11 @@ func TestDirFromModule_registry(t *testing.T) {
 	}
 
 	fixtureDir := filepath.Clean("testdata/empty")
-	dir, done := tempChdir(t, fixtureDir)
+	tmpDir, done := tempChdir(t, fixtureDir)
+	dir, err := filepath.EvalSymlinks(tmpDir)
```

Comment:
Could you add a little comment above here explaining why this is necessary? It's surprising that a `tempChdir` result would need `EvalSymlinks` applied to it but I'm sure there was a reason! :grinning: 

(Similarly for the other `EvalSymlinks` calls elsewhere here)

Reply:
Yes! Added a comment! 

For PR posterity: The module installer runs `filepath.EvalSymlinks` on the destination directory, and the result of that is what's returned by the install hooks. If you are on, for eg, a mac whose `TMPDIR` env var points to a symlinked directory, the tests - which compare expected installation dir with what's returned by the install hooks - would fail. 

I could also (easily) make the tests path by changing my machine's configured TMPDIR to a non-symlinked directory, but as I suspect this impacts most folks using mac osx and developing locally, I felt it was worth addressing in the test.